docs/hipify-clang.rst

```diff
@@ -1,32 +1,37 @@
+.. meta::
+   :description: Tools to automatically translate CUDA source code into portable HIP C++
+   :keywords: HIPIFY, ROCm, library, API, tool

```

Comment:
Here and in the rest of the changed files below:

`+` CUDA
`+` CUDA2HIP
`+` hipify-clang
`+` hipify-perl
`-` API

Reply:
Done!